### PR TITLE
Disabling Server header on Jetty

### PR DESCRIPTION
**Pull request by t-tsutsumi**
There is no point in returning the `Server` header.

For reference, Tomcat 9.0.x and 8.5.x no longer set the Server header by default.

### CODE REVIEW

**Comment by takezoe** [comment]
Conflicting with #1563.

**Comment by t-tsutsumi** [comment]
@takezoe Conflict was resolved.

File: src/main/java/JettyLauncher.java [hunk 1]
```diff
@@ -64,6 +68,11 @@ public static void main(String[] args) throws Exception {
 //        connector.setPort(port);
 //        server.addConnector(connector);
 
+        // Disabling Server header
+        Arrays.stream(server.getConnectors()).map(Connector::getConnectionFactories).flatMap(Collection::stream)
+                .filter(HttpConnectionFactory.class::isInstance).map(HttpConnectionFactory.class::cast)
+                .map(HttpConnectionFactory::getHttpConfiguration).forEach(config -> config.setSendServerVersion(false));
```
Comment (takezoe) [inline src/main/java/JettyLauncher.java]: I don't think that stream API is appropriate in this case. for loop like following is natural and much enough.
```java
for(Connector connector: server.getConnectors()){
    for(ConnectionFactory factory: connector.getConnectionFactories()){
        if(factory instanceof HttpConnectionFactory){
            ((HttpConnectionFactory) factory).getHttpConfiguration().setSendServerVersion(false);
        }
    }
}
```

Comment (t-tsutsumi) [inline src/main/java/JettyLauncher.java]: @takezoe It makes sense. I revised according to your instruction.

**Comment by takezoe** [comment]
@t-tsutsumi Thanks for quick fix!

**Pull request merged**